src/ensembl/src/content/app/browser/drawer/drawer-views/track-details/TrackDetails.scss

```diff
@@ -0,0 +1,51 @@
+@import 'src/styles/common';
+
+.trackName{
+    font-weight: $bold;
+    display: inline-block;
+}
```

Comment:
here and below — 2 spaces rather than 4 for indent.

Reply:
Changed it all to 2 spaces